torch/_inductor/codegen/cpp_micro_gemm.py

```diff
@@ -398,9 +417,20 @@ class CppMicroGemmFP32Vec(CppMicroGemm):
         }
 
         if constexpr (row == 0) {
-            {%- if input_dtype == torch.bfloat16 or input_dtype == torch.float16 %}
+            {%- if input2_dtype in [torch.bfloat16, torch.float16] %}
             auto b = VectorizedIn::loadu(B + k * ldb + col * VLEN, VLEN);
             vb[col] = at::vec::convert<{{compute_t}}>(b);
+            {%- elif input2_dtype == torch.int8 %}
+            // Load 64 int8 elements & convert them into 16 FP32 elements each in 4 vector registers
+            if constexpr (col % 4 == 0) {
+                constexpr auto remaining = std::min<int>(4, COLS - col);
+                {{kernel.unroll_pragma(4)}}
+                for (int idx = 0; idx <= remaining; idx++) {
```

Comment:
```suggestion
                for (int idx = 0; idx < remaining; idx++) {
```

Reply:
Oops! Thanks a lot for catching it! The original implementation had 3 as max, so this error must've crept in while I was copy-pasting.